packages/block-library/src/page-list-item/block.json

```
@@ -3,10 +3,10 @@
 	"apiVersion": 2,
```

Does this need the `parent` to be set to `core/page-list`?
```suggestion
	"apiVersion": 2,
	"parent": [ "core/page-list" ],
```

---

Isn't this set a few lines down?

---

Ok. Maybe my diff didn't show that 👍